app/views/markets/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>Farmer's Markets!</h1>
+
+<section>
+  <ul>
+    <% @all_markets.each do |market| %>
+      <% if market[:id] != market[:market_id] %>
```

Comment:
The markets table should already have an implicit id, so you probably don't need another column that saves another market_id that is the same! Although I'm not positive about the requirements of this project, so ignore that if it's wrong.  But regardless, try to keep as much logic out of the views as possible!